homeassistant/components/volvo/icons.json

```diff
@@ -1,5 +1,271 @@
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "fog_light_rear_warning": {
+        "default": "mdi:car-light-fog",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "hazard_lights_warning": {
+        "default": "mdi:hazard-lights",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "high_beam_left_warning": {
+        "default": "mdi:car-light-high",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "high_beam_right_warning": {
+        "default": "mdi:car-light-high",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "hood": {
+        "default": "mdi:car-door-lock",
+        "state": {
+          "on": "mdi:car-door-lock-open"
+        }
+      },
+      "low_beam_left_warning": {
+        "default": "mdi:car-light-dimmed",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "low_beam_right_warning": {
+        "default": "mdi:car-light-dimmed",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "oil_level_warning": {
+        "default": "mdi:oil-level"
+      },
+      "position_light_front_left_warning": {
+        "default": "mdi:car-parking-lights",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "position_light_front_right_warning": {
+        "default": "mdi:car-parking-lights",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "position_light_rear_left_warning": {
+        "default": "mdi:car-parking-lights",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "position_light_rear_right_warning": {
+        "default": "mdi:car-parking-lights",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "registration_plate_light_warning": {
+        "default": "mdi:lightbulb-outline",
+        "state": {
+          "on": "mdi:lightbulb-off-outline"
+        }
+      },
+      "reverse_lights_warning": {
+        "default": "mdi:car-light-dimmed",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "service_warning": {
+        "default": "mdi:wrench",
+        "state": {
+          "on": "mdi:wrench-clock"
+        }
+      },
+      "side_mark_lights_warning": {
+        "default": "mdi:wall-sconce-round",
+        "state": {
+          "on": "mdi:car-light-alert"
+        }
+      },
+      "sunroof": {
+        "default": "mdi:car-door-lock",
+        "state": {
+          "on": "mdi:car-door-lock-open"
+        }
+      },
+      "tailgate": {
+        "default": "mdi:car-door-lock",
+        "state": {
+          "on": "mdi:car-door-lock-open"
+        }
+      },
+      "tank_lid": {
+        "default": "mdi:car-door-lock",
+        "state": {
+          "on": "mdi:car-door-lock-open"
+        }
+      },
```

Comment:
There are a lot of sensors with a *door* icon. Would be nice to find some more suitable icons, but I guess you already did a search.